lib/components/right_drawer.dart

```diff
@@ -0,0 +1,239 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_inappwebview/flutter_inappwebview.dart';
+import 'package:provider/provider.dart';
+import 'package:rtchat/components/channel_search_dialog.dart';
+import 'package:rtchat/models/audio.dart';
+import 'package:rtchat/models/channels.dart';
+import 'package:rtchat/models/layout.dart';
+import 'package:rtchat/models/quick_links.dart';
+import 'package:rtchat/models/tts.dart';
+import 'package:rtchat/models/user.dart';
+import 'package:rtchat/screens/settings/quick_links.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class _ChannelPickerValue {
+  final Channel? channel;
+  final bool isAdd;
+
+  const _ChannelPickerValue({this.channel, this.isAdd = false});
+}
+
+class RightDrawer extends StatelessWidget {
+  final browser = ChromeSafariBrowser();
+
+  RightDrawer({Key? key}) : super(key: key);
+
+  void launchLink(QuickLinkSource source) async {
+    final isWebUrl =
+        source.url.scheme == 'http' || source.url.scheme == 'https';
+    if (isWebUrl) {
+      await browser.open(url: source.url);
+    } else {
+      await launch(source.url.toString());
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    print('rebuilding drawer');
+    return Drawer(
+      child: Column(
```

Comment:
Use a ListView instead of a column, see https://docs.flutter.dev/cookbook/design/drawer#3-populate-the-drawer-with-items